app/assets/stylesheets/partials/_cafe.scss

```diff
@@ -0,0 +1,30 @@
+.cafe {
+  &.working {
+    color: $fsek-orange;
+    display: block;
+    font-weight: bold;
+    margin: 20px 0 20px 0;
+  }
+
+  tr.with_worker {
+
+    td {
+      background-color: rgba($unify-green, 0.4) !important;
+    }
+  }
+
+  &.competition {
+    .worker {
+      display: inline-block;
+      font-size: 20px;
+
+      &.count {
+        &.right {
+          float: right;
+        }
+
+        color: $fsek-orange;
```

Comment:
Expected item on line 26 to appear before line 22. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets